locales/es.json

```diff
@@ -29,35 +29,35 @@
-  "footer-col-1-6": "<a href='https://www.freecodecamp.org/news/javascript-url-encode-example-how-to-use-encodeuricomponent-and-encodeuri/'>URL Encode</a>",
+  "footer-col-1-5": "<a href='https://www.freecodecamp.org/espanol/news/curso-debounce-javascript-como-hacer-que-tu-js-espere/'>Debounce</a>",
+  "footer-col-1-6": "<a href='https://www.freecodecamp.org/espanol/news/url-codificacion-como-utilizar-encodeuricomponent-javascript/'>URL Encode</a>",
   "footer-col-1-7": "<a href='https://www.freecodecamp.org/news/make-it-blink-html-tutorial-how-to-use-the-blink-tag-with-code-examples/'>Blink HTML</a>",
-  "footer-col-1-8": "<a href='https://www.freecodecamp.org/news/python-returns-multiple-values-how-to-return-a-tuple-list-dictionary/'>Python Tuple</a>",
-  "footer-col-1-9": "<a href='https://www.freecodecamp.org/news/javascript-array-insert-how-to-add-to-an-array-with-the-push-unshift-and-concat-functions/'>JavaScript Push</a>",
+  "footer-col-1-8": "<a href='https://www.freecodecamp.org/espanol/news/tuplas-listas-python/'>Python Tupla</a>",
+  "footer-col-1-9": "<a href='https://www.freecodecamp.org/espanol/news/como-agregar-elementos-a-un-arreglo/'>JavaScript Push</a>",
   "footer-col-1-10": "<a href='https://www.freecodecamp.org/news/java-list-tutorial-util-list-api-example/'>Java List</a>",
-  "footer-col-2-1": "<a href='https://www.freecodecamp.org/news/learn-ux-design-self-taught-user-experience-designer/'>UX</a>",
-  "footer-col-2-2": "<a href='https://www.freecodecamp.org/news/what-is-design-thinking-an-introduction-to-the-design-process-for-entrepreneurs-and-developers/'>Design Thinking</a>",
-  "footer-col-2-3": "<a href='https://www.freecodecamp.org/news/prime-numbers-list-chart-of-primes/'>Prime Numbers List</a>",
-  "footer-col-2-4": "<a href='https://www.freecodecamp.org/news/product-design-explained-in-plain-english/'>Product Design</a>",
+  "footer-col-2-1": "<a href='https://www.freecodecamp.org/espanol/news/diseno-ux-convertirse-en-un-disenador-de-experiencia-de-usuario/'>UX</a>",
+  "footer-col-2-2": "<a href='https://www.freecodecamp.org/espanol/news/pensamiento-de-diseno-emprendedores-desarrolladores/'>Pensamiento de Diseño</a>",
+  "footer-col-2-3": "<a href='https://www.freecodecamp.org/espanol/news/lista-de-numeros-primos-una-grafica-con-todos-los/'>Lista de Números Primos</a>",
+  "footer-col-2-4": "<a href='https://www.freecodecamp.org/espanol/news/product-design-explicado-en-un-espanol-sencillo/'>Diseño de Producto</a>",
   "footer-col-2-5": "<a href='https://www.freecodecamp.org/news/what-is-digital-design-and-why-does-it-matter/'>Digital Design</a>",
-  "footer-col-2-6": "<a href='https://www.freecodecamp.org/news/best-coding-games-online-adults-learn-to-code/'>Coding Games</a>",
+  "footer-col-2-6": "<a href='https://www.freecodecamp.org/espanol/news/los-mejores-juegos-online-para-aprender-programar/'>Juegos de Programación</a>",
```

Comment:
"Juegos de Programación" is just a bit too long. Is there another term or abbreviation we can use here to shorten this line?

Reply:
"Juegos" maybe. But it doesn't specify that is a "coding" game. Another translation is "Juegos de codificación"

Reply:
Yes, I agree. It would be best to get both those terms in there.

I tried "Juegos de Codificación" and it's just a bit too long. Would "Juegos de Código" work here?

Reply:
Yes, that works.